src/org/traccar/database/DataManager.java

```diff
@@ -115,6 +124,73 @@ private void initDatabase() throws Exception {
         }
     }
 
+    public static String constructObjectQuery(String action, Class<?> clazz, boolean additional) {
```

Comment:
Can you please explain what `additional` mean? Ideally with some examples. Also, maybe `extended` is better name (depends on what it's used for).